docs/src/pages/guides/styling.md

```diff
@@ -159,6 +162,15 @@ Now you're ready to write Tailwind! Our recommended approach is to create a `src
 
 As an alternative to `src/styles/global.css`, You may also add Tailwind utilities to individual `pages/*.astro` components in `<style>` tags, but be mindful of duplication! If you end up creating multiple Tailwind-managed stylesheets for your site, make sure you're not sending the same CSS to users over and over again in separate CSS files.
 
+#### v20 Tweaks to the tailwind import
+As of [Version 20](https://github.com/snowpackjs/astro/releases/tag/astro%400.20.0), Astro will no longer bundle, build and process public files. You now need to move your styles into the `src` directory and import them like so, making use of [Astro Resolve](http://localhost:3000/reference/api-reference#astroresolve): 
+```html
+  <link 
+    rel="stylesheet" 
+    href={`${Astro.resolve(../styles/global.css)}`} 
+  />
```

Comment:
Because this should be added in `.astro` file, not in JSX, you don't need self closing tag `/>`, but rather just `>`.

Reply:
Addressed: de85fce